intro-HPC/gent/macros.tex

```diff
@@ -3,20 +3,22 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
+\macro{\hpc}{UGent-HPC}
 \macro{\hpcname}{Delcatty}
 \macro{\loginnode}{login.hpc.ugent.be}
 \macro{\loginhost}{gligar01.gligar.gent.vsc}
 \macro{\operatingsystem}{Scientifc Linux 6.5}
-\macro{\tutorialdir}{/apps/gent/tutorials}
-\macro{\examplesdir}{/apps/gent/tutorials/Intro-HPC/examples}
+\macro{\tutorialdir}{/apps/antwerpen/tutorials}
+\macro{\examplesdir}{/apps/antwerpen/tutorials/Intro-HPC/examples}
```

Comment:
well, I checked and there is no `/apps/gent/tutorials`
please do provide me with the correct value here.